operations/system_configuration_service/Set_BMC_Credentials.md

```diff
@@ -2,6 +2,12 @@
 
 Use the System Configuration Service \(SCSD\) to set the BMCs credentials to unique values, or set them all to the same value. Redfish BMCs get installed into the system with default credentials. Once the machine is shipped, the Redfish credentials must be changed on all BMCs. This is done using System Configuration Service \(SCSD\) through the Cray CLI.
 
+**Important:** When changing credentials they may exist a device specific procedure that should be perfered over this one. List of device specific credential changing procedures:
```

Comment:
Change 

"When changing credentials they may exist a device specific procedure that should be perfered over this one. List of device specific credential changing procedures:"

to

"If the credentials for other devices need to be changed, refer to the following device-specific credential changing procedures:"

Reply:
Should be resolved fixed now.